src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy

```diff
@@ -138,7 +138,7 @@ class SimpleAndroidApp {
 
         file("gradle.properties") << """
                 android.useAndroidX=true
-                org.gradle.jvmargs=-Xmx2048m -Dkotlin.daemon.jvm.options=-Xmx1500m
+                org.gradle.jvmargs=-Xmx1536m -XX:+UseParallelGC -Dkotlin.daemon.jvm.options=-Xmx768m,-Xms256m,-XX:MaxMetaspaceSize=1g
```

Comment:
does it make sense that the maxmetaspacesize is greater than the heap size?

also regarding parallelGC, it might be more performant but it does not recliam memory as quickly as G1. does switching to parallel give other benefits opposed to G1?

Reply:
done https://github.com/gradle/android-cache-fix-gradle-plugin/pull/438/commits/005cccfe95885a270caf6eba977755fe2f201a2e

didn't observe any issues in the CI build, asking for review again